src/vue/common/EmailGetter.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <span
+    v-if="isLoaded"
+    class="email-getter"
+    :title="isTitled && (email || id)"
```

Comment:
incapsulate email and accountId and balanceId into a `result` data property

Reply:
Resolved, updated